tests/FixerFactoryTest.php

```diff
@@ -216,13 +216,18 @@ public function getFixersPriorityCases()
             array($fixers['binary_operator_spaces'], $fixers['align_double_arrow']), // tested also in: align_double_arrow,binary_operator_spaces.test
             array($fixers['binary_operator_spaces'], $fixers['align_equals']), // tested also in: align_double_arrow,align_equals.test
             array($fixers['class_definition'], $fixers['no_trailing_whitespace']), // tested also in: class_definition,no_trailing_whitespace.test
+            array($fixers['no_unused_imports'], $fixers['no_extra_consecutive_blank_lines']), // tested also in: no_unused_imports,no_extra_consecutive_blank_lines.test
+            array($fixers['single_import_per_statement'], $fixers['no_unused_imports']), // tested also in: single_import_per_statement,no_unused_imports.test
+            array($fixers['no_leading_import_slash'], $fixers['ordered_imports']), // tested also in: no_leading_import_slash,ordered_imports.test
+            array($fixers['no_extra_consecutive_blank_lines'], $fixers['ordered_imports']),
+            array($fixers['no_unused_imports'], $fixers['no_leading_import_slash']),
+            array($fixers['single_import_per_statement'], $fixers['no_leading_import_slash']),
```

Comment:
let me check, I started this branch before the big renaming of fixer so this could be a merge gone wrong.